Marlin/ultralcd.cpp

```diff
@@ -2315,9 +2339,9 @@ void kill_screen(const char* lcd_msg) {
     void _reset_e_acceleration_rate(const uint8_t e) { if (e == active_extruder) _reset_acceleration_rates(); }
     void _reset_e0_acceleration_rate() { _reset_e_acceleration_rate(0); }
     void _reset_e1_acceleration_rate() { _reset_e_acceleration_rate(1); }
-    #if E_STEPPERS > 2
+    #if EXTRUDERS > 2
```

Comment:
there are many Labels that don't really fit to current possible configurations. 
The switching extruder have 2 extruder with only 1 stepper but it is can be usefull to define another stepping parameter for the 2 extruder (i have flexible filament, that need more stepps fore the same extrude length, cause it is squeezed by the spring of the extruder)
If you can say me with parameter i should can use, i will change it.